exercises/hamming/HammingSpec.groovy

```diff
@@ -0,0 +1,50 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class HammingSpec extends Specification {
```

Comment:
Perhaps it's mentioned somewhere else (in the exercise?), but if I only look at this spec I wouldn't know (the overview of) what a Hamming-distance is and what the letters stand for